tools/fpgainfo/fmeinfo.c

```diff
@@ -82,10 +180,11 @@ fpga_result fme_command(fpga_token *tokens, int num_tokens, int argc,
 	(void)argv;
 
 	fpga_result res = FPGA_OK;
-
+	int verbose_opt = 0;
 	optind = 0;
 	struct option longopts[] = {
 		{"help", no_argument, NULL, 'h'},
+		{"verbose", no_argument, NULL, 'v'},
```

Comment:
Can we use V here instead of v (version)?

Reply:
verbose changes to V